<ui version="4.0" >
 <class>MsgSend</class>
 <widget class="QWidget" name="MsgSend" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>messageSend</string>
  </property>
  <property name="styleSheet" >
   <string notr="true" />
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="Ac_Status" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="textFormat" >
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="MsgBrowser" >
         <property name="focusPolicy" >
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="styleSheet" >
          <string notr="true" />
         </property>
         <property name="lineWidth" >
          <number>2</number>
         </property>
         <property name="verticalScrollBarPolicy" >
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="autoFormatting" >
          <set>QTextEdit::AutoAll</set>
         </property>
         <property name="tabChangesFocus" >
          <bool>false</bool>
         </property>
         <property name="undoRedoEnabled" >
          <bool>true</bool>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="html" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="overwriteMode" >
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags" >
          <set>Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="" >
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QLabelMouse" name="TB_CHANGEMOD" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>change send mode</string>
         </property>
         <property name="text" >
          <string>Change send mode</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLabelMouse" name="TB_HISTROY" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>look histroy</string>
         </property>
         <property name="text" >
          <string>histroy</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLabelMouse" name="TB_NUDGE" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>nudge</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <spacer>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>227</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4" >
        <widget class="QLabel" name="input_nm" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="5" >
        <widget class="QLabelMouse" name="TB_SEND" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>send</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6" >
        <widget class="QLabelMouse" name="TB_FACE" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Face</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="7" >
        <widget class="QTextEdit" name="MsgEdit" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet" >
          <string notr="true" />
         </property>
         <property name="tabChangesFocus" >
          <bool>false</bool>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLabelMouse</class>
   <extends>QLabel</extends>
   <header>fxLableMouse.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
